ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/sse/ClientServerSentEventsSession.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2014-2023 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.client.plugins.sse
+
+import io.ktor.sse.*
+import kotlinx.coroutines.*
+import kotlinx.coroutines.channels.*
+
+/**
+ * A client Server-sent events session.
+ */
+public interface ClientServerSentEventsSession : CoroutineScope {
```

Comment:
Could we name it `ClientSSESession`? `SSE` is a well-known keyword; it can be searched. Is it possible to have an SSE session for the server in the future?

Reply:
Thanks, I will rename it and related things. Yes, it is possible to have an SSE session for the server in the future